src/Microsoft.Docs.MarkdigExtensions/MarkdownExtensions.cs

```diff
@@ -139,9 +139,9 @@ public static MarkdownPipelineBuilder UseNestedColumn(this MarkdownPipelineBuild
             return pipeline;
         }
 
-        public static MarkdownPipelineBuilder UseTripleColon(this MarkdownPipelineBuilder pipeline, MarkdownContext context)
+        public static MarkdownPipelineBuilder UseTripleColon(this MarkdownPipelineBuilder pipeline, MarkdownContext context, TripleColonExtensionType enabledExtensionTypes = (TripleColonExtensionType)~0)
```

Comment:
Changing this method signature breaks library contract and could potentially break libraries depending on it. Add a new method that can opt-in to triple colon extensions individually. We also don't need an enum.

```csharp
public static MarkdownPipelineBuilder UseTripleColon<T>(this MarkdownPipelineBuilder pipeline, MarkdownContext context, T extension) where T : ITripleColonExtensionInfo
```